examples/kuka_rsi_robot_tests/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>kuka_rsi_robot_tests</name>
```

Comment:
I think `kuka_mobile_robot_examples` would be better, as this includes examples for mobile platform + robot integration rather than tests

Reply:
actually, these tests are on the rsi_robot,(only robot movements), not platform movements. The platform tests, i have in the driver package directly

Reply:
okay sorry, you are right, these are indeed robot tests. 
However I cannot see any RSI-specific commands, so they are rather general tests/examples for kuka robotic arms, are they not? In this case, `kuka_driver_movement_examples`/ `kuka_driver_movement_tests` or something similiar might be more appropriate

Reply:
yes, you are correct, the tests are not specific to rsi

Reply:
Sorry, actually, the test is specific to rsi because, it's startup sequence has the activation of the rsi driver.

Reply:
But the activation sequence is the same for all KUKA drivers, so I would say it is not RSI specific